Scripts/RemediationScripts/Remediate-CdnFrontDoorMinTLSVersion.ps1

```diff
@@ -0,0 +1,1094 @@
+                        $customDomainResponse = Invoke-WebRequest -Method GET -Uri $uri -Headers $headers -UseBasicParsing
+                    
+                        if($customDomainResponse.StatusCode -ge 200 -and $customDomainResponse.StatusCode -le 399)
+                        {
+                            if($null -ne $customDomainResponse.Content)
+                            {
+                                $custom = $customDomainResponse.Content | ConvertFrom-Json  
+                                if($null -ne $custom)
+                                {
+                                    $TypeOfCertificate = $custom.properties.tlsSettings.certificateType
+                                    $MinTLSVersion = $custom.properties.tlsSettings.minimumTlsVersion
+                                }
+                            }
+                        }        
+                    }
+                    if($null -ne $MinTLSVersion)
+                    {
+                        if($MinTLSVersion -ne $requiredMinTLSVersion)
+                        {
+                            $frontDoorEp.DomainName = $domainName
+                            $frontDoorEp.FrontDoorName = $resourceName
+                            $frontDoorEp.MinTLSVersion = $MinTLSVersion
+                            $frontDoorEp.ResourceGroup = $resourceGroupName
+                            $frontDoorEp.TypeOfCertificate = $TypeOfCertificate
+                            $frontDoorEp.isMinTLSVersionSetOnCustomDomain = $false
+                        }
+                        else
+                        {
+                            $frontDoorsEndpointWithtReqMinTLSVersion = $item
+                        }
+                    }
+                    if($null -ne $frontDoorEp)
+                    {
+                       $frontDoorsEndpointWithoutReqMinTLSVersion =  $frontDoorEp | Select-Object @{N='ResourceGroupName';E={$frontDoorEp.ResourceGroup}},
+                                                                                    @{N='ResourceName';E={$frontDoorEp.FrontDoorName}},
+                                                                                    @{N='DomainName';E={$frontDoorEp.DomainName}},
+                                                                                    @{N='CertificateType';E={$frontDoorEp.TypeOfCertificate}},
+                                                                                    @{N='MinimumTlsVersion';E={$frontDoorEp.MinTLSVersion}},
+                                                                                    @{N='isMinTLSVersionSetOnCustomDomain';E={$frontDoorEp.isMinTLSVersionSetOnCustomDomain}}
+                    }
+                    
+                    else
+                    {
+                        Write-Host "Skipping this Front End Point..." -ForegroundColor $([Constants]::MessageType.Warning)
```

Comment:
Here skipping means it's complaint ryt?

Reply:
No, Actually It is not having custom domain endpoint this is why we are skipping.